Rubberduck.VBEEditor/Extensions/VbeExtensions.cs

```diff
@@ -62,11 +62,11 @@ public static IHostApplication HostApplication(this VBE vbe)
                 switch (reference.Name)
                 {
                     case "Excel":
-                        return new ExcelApp();
+                        return new ExcelApp(vbe);
                     case "Access":
                         return new AccessApp();
```

Comment:
This change looks like it's going to break the Access, PowerPoint, & Publisher hosts. Can we just call it quits on this? Would it be nice to support CorelDraw? Yeah, is it worth making changes that could break many more existing users? No.